### ISSUE TITLE
failed to allocate second dataset buffer

### ISSUE THREAD

**Issue by Allvasound** [trexminer/T-Rex#1010]
Hello.

Have issue with 3080 LHR. failed to allocate second dataset buffer
With 3080 TI LHR no issue with second dataset buffer.

I mining ergo eth

newest trex version

windows 10

ram 8 gb

amd ryzen 5 1600

256 gb ssd

msi b 450 a pro max with newest bios version

Virtual memory set it to 60000 for 6 3080 LHR, try to set 70000, same issue

nvidia driver 272.12. work best for me

6 gpu‘s on one morherboard with 6 pcie slots

Settings: core 0, memory 1575, Lock Clock by 1575, 725 mv

i hope anyone can help me.

**Comment by trexminer** [comment]
`failed to allocate second dataset buffer` is not an error, just a warning. It tells you that your GPU can't work in dual memory buffer mode because it doesn't have enough VRAM, so it falls back to single buffer mode. See `--dataset-mode` in readme for details.

**Comment by Allvasound** [comment]
@trexminer Hello thank you for your answer. can you tell me where i find the readme?

**Comment by trexminer** [comment]
Yes. See my original response for the explanation as to why it's happening and what it is.
